python/tox.ini

```diff
@@ -1,8 +1,8 @@
 [tox]
-envlist = py27
+envlist = py27, py35, py37
```

Comment:
One thing to keep in mind with this is that typing "make python" will not succeed if all three environments are not present.

Is this okay?  or should we allow tox to succeed if some combination of environments are there or at least one environment is there?  

`
Running Python tests ...

cd /Users/dzollo/source/libsbp/python/ && tox -e py27,py35,py37
GLOB sdist-make: /Users/dzollo/source/libsbp/python/setup.py
py27 inst-nodeps: /Users/dzollo/source/libsbp/python/.tox/dist/sbp-2.4.10.dev9+g08c2ed72.zip
py27 installed: -f file:///Users/dzollo/.cache/pip/wheelhouse,certifi==2018.11.29,chardet==3.0.4,construct==2.9.33,cov-core==1.15.0,coverage==4.4.1,filelock==3.0.10,futures==3.2.0,httpretty==0.9.4,idna==2.8,pluggy==0.8.1,py==1.7.0,pyftdi==0.13.4,pylibftdi==0.17.0,pyserial==3.4,pytest==3.1.2,pytest-cov==2.5.1,pyusb==1.0.2,requests==2.21.0,requests-futures==0.9.9,ruamel.ordereddict==0.4.13,ruamel.yaml==0.15.86,sbp==2.4.10.dev9+g08c2ed72,six==1.12.0,toml==0.10.0,tox==3.7.0,urllib3==1.24.1,virtualenv==16.2.0
py27 runtests: PYTHONHASHSEED='1118137729'
py27 runtests: commands[0] | py.test -v tests/
=============================================================================== test session starts ===============================================================================
platform darwin -- Python 2.7.15, pytest-3.1.2, py-1.7.0, pluggy-0.4.0 -- /Users/dzollo/source/libsbp/python/.tox/py27/bin/python
cachedir: .cache
rootdir: /Users/dzollo/source/libsbp/python, inifile: pytest.ini
plugins: cov-2.5.1
collected 25 items 

tests/sbp/test_messages.py::test_messages PASSED
tests/sbp/test_table.py::test_table_count PASSED
tests/sbp/test_table.py::test_table_unqiue_count PASSED
tests/sbp/test_table.py::test_available_messages PASSED
tests/sbp/client/test_driver.py::test_http_test_pass_streaming <- .tox/py27/lib/python2.7/site-packages/httpretty/core.py PASSED
tests/sbp/client/test_driver.py::test_http_test_pass_retry <- .tox/py27/lib/python2.7/site-packages/httpretty/core.py PASSED
tests/sbp/client/test_driver.py::test_http_test_pass <- .tox/py27/lib/python2.7/site-packages/httpretty/core.py PASSED
tests/sbp/client/test_driver.py::test_http_test_fail <- .tox/py27/lib/python2.7/site-packages/httpretty/core.py PASSED
tests/sbp/client/test_driver.py::test_tcp_logger PASSED
tests/sbp/client/test_handler.py::test_framer_receive_empty PASSED
tests/sbp/client/test_handler.py::test_framer_receive_bad_preamble PASSED
tests/sbp/client/test_handler.py::test_framer_bad_crc PASSED
tests/sbp/client/test_handler.py::test_framer_ok PASSED
tests/sbp/client/test_handler.py::test_handler_callbacks PASSED
tests/sbp/client/test_handler.py::test_multiple_handler_callbacks PASSED
tests/sbp/client/test_handler.py::test_child_iter PASSED
tests/sbp/client/test_handler.py::test_filter PASSED
tests/sbp/client/test_handler.py::test_dead_gc PASSED
tests/sbp/client/test_handler.py::test_late_iter PASSED
tests/sbp/client/test_logger.py::test_log PASSED
tests/sbp/client/test_logger.py::test_json_log PASSED
tests/sbp/client/test_logger.py::test_non_utf8_json_log PASSED
tests/sbp/client/test_logger.py::test_msg_print xfail
tests/sbp/client/test_logger.py::test_udp_logger PASSED
tests/sbp/client/test_logger.py::test_rolling_json_log PASSED

====================================================================== 24 passed, 1 xfailed in 9.25 seconds =======================================================================
py35 create: /Users/dzollo/source/libsbp/python/.tox/py35
ERROR: InterpreterNotFound: python3.5
py37 create: /Users/dzollo/source/libsbp/python/.tox/py37
ERROR: InterpreterNotFound: python3.7
_____________________________________________________________________________________ summary _____________________________________________________________________________________
  py27: commands succeeded
ERROR:  py35: InterpreterNotFound: python3.5
ERROR:  py37: InterpreterNotFound: python3.7`

Reply:
IMO this is a reasonably good arrangement. If a developer doesn't want to run tests for all python versions, he can limit the runs by specifying an environment variable but by default all supported versions are tested, which also applies to travis runs.

Reply:
lgtm thanks!